src/core/mount.c

```diff
@@ -1734,6 +1735,8 @@ static void mount_enumerate(Manager *m) {
                         goto fail;
                 }
 
+                m->mount_last_read_usec = 0;
+                m->mount_last_duration_usec = 0;
```

Comment:
this should be necessary. All our objects when allocated are memset() to zero. Hence the Manager object is too